Preference Access SSN Matching/LETTERSPECS/PREFACCESS.SSNMATCHING.CFG

```diff
@@ -0,0 +1,155 @@
+* INVALID TARGET ACCOUNT, SHARE, LOAN AND EXTERNAL LOAN TYPES
+*------------------------------------------------------------------------------
+*  Line must begin with "ITAT:" (for accounts)
+*  Line must begin with "ITST:" (for shares)
+*  Line must begin with "ITLT:" (for loans)
+*  Line must begin with "ITET:" (for external loans)
+*
+*  Enter the Account, Share, Loan and or External Loan types which will exclude
+*  an Account, Share, Loan and/or External Loan from accessibility by this
+*  program. If left blank, Account, Share, Loan types and/or External Loan
+*  types will not be a consideration when analyzing target accounts.
+*
+*  VALID VALUES: range of values from 0-9999 (Account, Share, Loan)
+*                range of values from 1-8 (External Loan)
+*  DEFAULT: (left blank - Account, Share, Loan and/or External Loan
+*            types not a factor)
+ITAT:
+ITST:
+ITLT:
+ITET:
+*
+*------------------------------------------------------------------------------
+* EXCLUDE ACCOUNTS WITH INVALID SSNs
+*------------------------------------------------------------------------------
+*  Line must begin with "ISSN:"
+*
+*  Enter a list of invalid primary name record SSNs. List as a single
+*  comma-delimited list. Accounts where the primary member's SSN are
+*  found in this list will be skipped over. Enter up to 10 different
+*  SSNs. Use only numeric values.
+*
+*  VALID VALUES: Up to 10, comma delimited, 9-digit numeric values
+*  DEFAULT: [left blank - no invalid SSNs]
+ISSN:000000000,111111111,222222222,333333333,444444444,555555555,666666666,777777777,888888888,999999999
+*------------------------------------------------------------------------------
+* EXCLUDE SHARES WITH A NON-ZERO IRS CODE
+*------------------------------------------------------------------------------
+*  Line must begin with "ENZS:"
+*
+*  If TRUE, or left blank, target Shares with the SHARE:IRSCODE value>0
+*  will be excluded from analysis.
+*
+*  VALID VALUES: TRUE or FALSE
+*  DEFAULT: TRUE
+ENZS:TRUE
+*
+*------------------------------------------------------------------------------
+* VALID TARGET NAME TYPES
+*------------------------------------------------------------------------------
+* Line must begin with "VTNT:"
+*
+* Enter the valid name types the program should check on the targeted
+* accounts.
+*
+*  VALID VALUES: Range of values from 0-99 or 'ALL' (at creation,
+*                the max name type is 53, but this will allow for
+*                new name types in the future.
+*  DEFAULT: (left blank or 'ALL' - all name types will be checked)
+VTNT:ALL
+*
+*------------------------------------------------------------------------------
+* PREFERENCE ACCESS FLAGS TO SET
+*------------------------------------------------------------------------------
+* When a Preference Access record is created, which of these flags
+* should be set? At least one must be set to TRUE. If left blank, FALSE
```

Comment:
In the comment for PAWF,  PADF and PAIF it says "if left blank, FALSE";
However, at the bottom of the comments it says DEFAULT: TRUE.
Which is it? Pick one and go with it. That way what is in the Config Worksheet will match what the code does for Default/Left Blank.

I suggested making them All TRUE for blank by Default. But that is just my opinion. 
Note: If you make Blank/Default TRUE, the PowerOn code will need to be changed.

Reply:
Restructured so the default is "TRUE", Specfile updated accordinglty.